models/v3/issueAssetRequest.js

```diff
@@ -0,0 +1,71 @@
+exports.models = {
```

Comment:
why was this file added?... If the only change is in `flags` - I think it can still be remain in v2